selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/assistant/CheckOpenedDialogThroughFindActionTest.java

```diff
@@ -22,7 +22,7 @@
 import org.testng.annotations.Test;
 
 /** @author Musienko Maxim */
-public class CheckOpendDialogThroughFindActionTest {
+public class CheckOpenedDialogThroughFindActionTest {
```

Comment:
Please, include this test into the CheSuite.xml to have it tested on daily propose.

Reply:
@dmytro-ndp This test in include to the CheSuite.xml

Reply:
I can't find it here https://github.com/eclipse/che/blob/5d3ddcb9910a0d64b3cbafd5c589a43af0d7fc90/selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml in your branch.

Reply:
@musienko-maxim: can you, please, look at this test and confirm that it should be include in the daily testing scope?